src/components/CurvedBottomBar/Readme.md

```diff
@@ -0,0 +1,177 @@
+### Example
```

Comment:
1. Keep readme as short as possible
2. Declare props of component in table structure with 3 columns i.e.
      - Prop -> Name of prop
      - Type -> Type of prop
      - Description -> Description related to prop. 
      
Please checkout root readme file for example. Use any online markdown editor.

Reply:
Regarding adding to App.tsx, we need to create a whole navigation structure with a bottom bar in this project. I think that would be a good idea as we will be using it as NPM package.